File: Lib/FPVR/Interfaces/IBarometerInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FPVR
{
    /// <summary>
    /// Provides information from a barometric sensor.
    /// </summary>
    public interface IBarometerInfo
    {
        /// <summary>
        /// The altitude in meters.
        /// </summary>
        double Altitude { get; }

        /// <summary>
        /// The vertical speed in meters per second.
        /// </summary>
        /// <remarks>
        /// This is also known as climb rate.
        /// </remarks>
        double VerticleSpeed { get; }
    }
}

File: Android/FPVR/app/src/main/java/com/solersoft/fpvr/fpvrlib/IBarometerInfo.java
package com.solersoft.fpvr.fpvrlib;

/**
 * Provides information from a barometric sensor.
 */
public interface IBarometerInfo
{
    //region Public Properties
    /**
     * Gets the altitude in meters.
     * @return The altitude in meters.
     */
    public double getAltitude();

    /**
     * Gets the vertical speed in meters per second. This is also known as climb rate.
     * @return The vertical speed in meters per second.
     */
    public double getVerticleSpeed();
    //endregion
}
